src/main/java/TTT/controller/UserController.java

```diff
@@ -0,0 +1,26 @@
+package TTT.controller;
+
+
+import TTT.users.CustomUser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class UserController {
```

Comment:
Why is user controller handling both main page and user details?

Reply:
My bad. Corrected!